shadow-agent/src/renderer/styles.css

```
@@ -1,28 +1,63 @@
 :root {
   color-scheme: dark;
-  font-family: "Segoe UI Variable Text", "Segoe UI", "Avenir Next", system-ui, sans-serif;
+  font-family: 'Geist', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)